File: app/src/main/java/com/mmjang/ankihelper/data/content/ContentDatabaseHelper.java
package com.mmjang.ankihelper.data.content;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

import com.mmjang.ankihelper.data.database.DBContract;
import com.mmjang.ankihelper.data.database.DatabaseContext;
import com.mmjang.ankihelper.util.Constant;

import java.io.File;

public class  ContentDatabaseHelper extends SQLiteOpenHelper {

    //private static final String DB_NAME = "ankihelper.db";
    private static final int VERSION = 1;
    private Context mContext;

    public ContentDatabaseHelper(Context context, String dbName) {
        super(new DatabaseContext(context),
                Constant.STORAGE_CONTENT_SUBDIRECTORY + File.separator + dbName, null, VERSION);
        mContext = context;
    }

    @Override
    public void onCreate(SQLiteDatabase db) {

    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/lmyby/ankihelper/data/content/ContentDatabaseHelper.kt
package com.lmyby.ankihelper.data.content

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper
import com.lmyby.ankihelper.data.database.DatabaseContext
import com.lmyby.ankihelper.util.Constant
import java.io.File

/**
 * Converted to Kotlin as part of Phase 2 data model migration
 */
class ContentDatabaseHelper(
    context: Context,
    dbName: String
) : SQLiteOpenHelper(
    DatabaseContext(context),
    Constant.STORAGE_CONTENT_SUBDIRECTORY + File.separator + dbName,
    null,
    VERSION
) {
    private val mContext: Context = context

    override fun onCreate(db: SQLiteDatabase) {
        // Empty implementation
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        // Empty implementation
    }

    companion object {
        private const val VERSION = 1
    }
}
